docs/config.md

```diff
@@ -204,6 +204,19 @@ motionSolverLibs    ("libfvMotionSolvers.so");
 solver              displacementLaplacian;
 ```
 
+#### FF
+
+The fluid-fluid coupling module supports reading and writing pressure, velocity, and their gradients.
```

Comment:
It would make sense to spell out the names, as we encoded them in the adapter, as their spelling actually matters for configuration purposes.

Reply:
Fixed, thanks!